code/modules/vehicle/vehicle.dm

```diff
@@ -138,10 +138,9 @@
 	handle_vehicle_offsets()
 
 /obj/vehicle/bullet_act(obj/item/projectile/Proj)
-	if(has_buckled_mobs())
-		for(var/m in buckled_mobs)
-			var/mob/living/buckled_mob = m
-			buckled_mob.bullet_act(Proj)
```

Comment:
You can have more than one mob buckled to a vehicle, in theory.

As for what is written here---I'm not sure why it would rectify the issue unless bullets are calling `Bump` multiple times...

Reply:
What happened previously was that this section of code made the projectile hit each/any passenger, but due to recent vehicle changes it became redundant and caused projectiles to hit passengers twice.